ntoskrnl/io/pnpmgr/pnpmgr.c

```diff
@@ -1760,6 +1765,80 @@ IopGetParentIdPrefix(PDEVICE_NODE DeviceNode,
     return Status;
 }
 
+static
+BOOLEAN
+IopValidateID(
+    _In_ PWCHAR Id,
+    _In_ SIZE_T MaxIdLen,
+    _In_ ULONG MaxSeparators,
+    _In_ BOOLEAN IsMultiSz)
+{
+    PWCHAR PtrSymbol;
+    PWCHAR StringEnd;
+    WCHAR Symbol;
```

Comment:
The right term is "character" or "char" for short.

Reply:
Changed